tftp/test_tftp.py

```diff
@@ -166,11 +167,165 @@ def test_bootcode(self):
         except Exception as e:
             self.fail(e)
         finally:
+            os.remove("./bootcodeTest.bin")
+            tftp_thread.stop()
+
+    def test_512(self):
```

Comment:
it looks like you have duplicated code here. can you pull that out into a function? (duplicated code is hard to maintain!)

Reply:
I modified it to use 1 test with different parameters